package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "energinet-datahub",
-  "version": "0.0.0",
+  "version": "1.0.0-chore-watt-npm-package.1710839337619",
```

Comment:
question: is this just out of sync, shouldn't the executor result in version `0.0.0-chore-watt-npm-package.1710839337619`?